connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoWriter.scala

```diff
@@ -155,9 +159,10 @@ object KustoWriter {
     KDSU.logInfo(myName, s"Ingesting from blob - partition: ${TaskContext.getPartitionId()} requestId: '${writeOptions.requestId}' $batchIdForTracing")
 
     tasks.asScala.foreach(t => try {
-      Await.result(t, KCONST.DefaultIngestionTaskTime)
+      Await.result(t, KCONST.DefaultMaximumIngestionTime)
+      // ALTHOUGH WE SAW THAT THE DM DIDNT EVEN GET THE MESSAGE EVENTUALLY
```

Comment:
This looks like a note-to-self more than a comment intended for the public to see.